src/EnergyPlus/DataZoneControls.hh

```diff
@@ -55,6 +55,7 @@
 #include <EnergyPlus/Data/BaseData.hh>
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/ZoneTempPredictorCorrector.hh>
```

Comment:
I'm a little concerned about a Data module including a manager module.  I get the feeling that something in the ZTPC header probably needs to be pulled here instead.  Not necessarily a problem for now if it is building alright, but just something to think about.